src/kwsbin/compute-atwv.cc

```diff
@@ -37,13 +37,31 @@ int main(int argc, char *argv[]) {
 
     const char *usage = "Computes the Actual Term-Weighted Value and prints it."
         "\n"
-        "Usage: compute-atwv [options]  ref-rspecifier hyp-rspecifier [alignment csv]\n"
-        " e.g.: compute-atwv ark:ref.1 ark:hyp.1 ali.csv\n"
+        "Usage: compute-atwv [options] <nof-trials> <ref-rspecifier> <hyp-rspecifier> [alignment-csv]\n"
+        " e.g.: compute-atwv 32485.4 ark:ref.1 ark:hyp.1 ali.csv\n"
         "\n"
-        "where the alignment format is compatible with the alignment produced\n"
-        "using the F4DE tool -- you are responsible for mapping the utterance\n"
-        "identifiers and the term string to the correct ones - use the script\n"
-        "utils/int2sym.pl and the utterance/keyword maps\n";
+        "NOTES: \n"
+        "  a) the number of trials is usually equal to the size of the searched\n"
+        "     collection in seconds\n"
+        "  b  the ref-rspecifier/hyp-rspecifier are the kaldi IO specifiers for both\n"
+        "     the reference and the hypotheses (found hits), consecutively.\n"
+        "     The format is the same for both of them. Each line is of \n"
+        "     the following format\n"
+        "\n"
+        "     <KW-ID> <utterance-id> <start-time-frames> <end-time-frames> <score>\n\n"
+        "     e.g.:\n\n"
+        "     KW106-189 348 459 560 0.8\n"
+        "\n"
+        "  b) the alignment-csv file format is equivalent with the alignment file\n"
+        "     produced using the F4DE tool. However, we do not set some fields\n"
+        "     and the utterance identifiers are numeric. You can use the script\n"
+        "     utils/int2sym.pl and the utterance/keyword maps to convert\n" 
+        "     the numerical ids into text\n"
```

Comment:
That is the issue -- a single example won't do because the format is quite complex and I don't think it's the right idea to explain it here. It would fit into the doxygen docs but there is no chance I can spend a lot of time on that right now :/
Also, it's not a mandatory parameter and can be ommitted (I'll mention that explicitly).